ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -332,6 +335,20 @@ - (void)updateCacheToItem:(ADTokenCacheItem *)cacheItem
     [[ADTelemetry sharedInstance] stopEvent:telemetryRequestId event:event];
 }
 
+- (BOOL)addOrUpdateItem:(nonnull ADTokenCacheItem *)item
+                context:(id<ADRequestContext>)context
+                  error:(ADAuthenticationError * __nullable __autoreleasing * __nullable)error
+{
+    NSURL *oldAuthority = [NSURL URLWithString:item.authority];
+    NSURL *newAuthority = [[ADAuthorityValidation sharedInstance] cacheUrlForAuthority:oldAuthority context:context];
+    
+    item.authority = [newAuthority absoluteString];
+    BOOL ret = [_dataSource addOrUpdateItem:item correlationId:context.correlationId error:error];
+    item.authority = [oldAuthority absoluteString];
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)